cmd/harmony.go

```diff
@@ -142,7 +167,16 @@ func main() {
 		ldb, _ = InitLDBDatabase(*ip, *port)
 	}
 
-	host := p2pimpl.NewHost(selfPeer)
+	host, err := p2pimpl.NewHost(&selfPeer)
+	if err != nil {
+		panic("unable to new host in harmony")
+	}
+
+	fmt.Printf("[Harmony] selfPeer: %v\n", selfPeer)
```

Comment:
is it debug message? log.Debug could be better?

Reply:
debug message, removed.